backend/oasst_backend/prompt_repository.py

```diff
@@ -388,12 +411,12 @@ def insert_reaction(self, task_id: UUID, payload: db_payload.ReactionPayload) ->
     def store_text_labels(self, text_labels: protocol_schema.TextLabels) -> TextLabels:
         model = TextLabels(
             api_client_id=self.api_client.id,
+            message_id=UUID(text_labels.message_id),
```

Comment:
if we require that the sent message id it always the read server message UUID we should change the type to `TextLabels.message_id` to UUID.

Reply:
Have changed this to UUID